instrumentation/delayed_job/lib/opentelemetry/instrumentation/delayed_job/middlewares/tracer_middleware.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'delayed/plugin'
+
+module OpenTelemetry
+  module Instrumentation
+    module DelayedJob
+      module Middlewares
+        # Delayed Job plugin that instruments invoke_job and other hooks
+        class TracerMiddleware < Delayed::Plugin
+          class << self
+            def instrument_enqueue(job, &block)
+              return block.call(job) unless enabled?
+
+              attributes = build_attributes(job)
+              attributes['messaging.operation'] = 'send'
+              tracer.in_span("delayed_job.#{job_queue(job)}.send", attributes: attributes, kind: :producer) do |span|
```

Comment:
```suggestion
              tracer.in_span("#{job_queue(job)} send", attributes: attributes, kind: :producer) do |span|
```
Based on [Span name semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#span-name).